.github/pull_request_template.md

```diff
@@ -17,6 +17,8 @@ Please read the [contributor agreements](https://github.com/optuna/optunahub-reg
 
 ## TODO List towards PR Merge
 
+If you introduce a new package in this PR, please check the following TODO list.
```

Comment:
```suggestion
## TODO List towards PR Merge

Please remove this section if this PR is not an addition of a new package.
Otherwise, please check the following TODO list:

```